src/NumericalAlgorithms/Spectral/Clenshaw.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#pragma once
+
+#include <cmath>
+#include <numeric>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace Spectral {
+/// Clenshaw's algorithm for evaluating linear combinations of
+/// basis functions obeying a given three term recurrence relation.
+/// See numerical recipes 3rd edition 5.4.2 for details.
+/// `alpha` and `beta` define the recursion
+/// \f$ f_n(x) = alpha f_{n-1}(x) + beta f_{n-2}(x) \f$
```

Comment:
`\alpha` and `\beta`

Reply:
Sorry, I meant in `\f$ f_n(x) = alpha f_{n-1}(x) + beta f_{n-2}(x) \f$`.  The ones on the line before can be whichever you prefer.